demo/campaign/src/common/blocks/CommonImageBlock.tsx

```diff
@@ -0,0 +1,63 @@
+import { calculateInheritAspectRatio, generateImageUrl } from "@comet/cms-site";
+import { MjmlImage, MjmlStyle } from "@luma-team/mjml-react";
+import { PixelImageBlockData } from "@src/blocks.generated";
+import { css } from "@src/util/stylesHelper";
+import { theme } from "@src/util/theme";
+import * as React from "react";
+
+import { getDamAllowedImageWidth } from "../helpers/imageBlockHelpers";
+
+interface Props extends React.ComponentProps<typeof MjmlImage> {
+    data: PixelImageBlockData;
+    desktopRenderWidth: number;
+}
+
+export const commonImageBlockStyles = (
+    <MjmlStyle>{css`
+        .image-block {
+            padding: 0 !important;
+        }
+
+        .image-block > table > tbody > tr > td {
+            width: 100% !important;
+        }
+
+        .image-block img {
+            height: auto !important;
+        }
```

Comment:
CSS will not be loaded on all mail-clients, the styling needs to work in that case as well, at least the desktop-variant. Please only use CSS for mobile/responsive styling and use inline-styles for everything else.